examples/run-cluster.sh

```diff
@@ -34,11 +34,11 @@ cleanup() {
 
 trap cleanup EXIT
 rm -rf /tmp/nats
-./marmot -config examples/node-1-config.toml -cluster-addr localhost:4221 -cluster-peers 'nats://localhost:4222/' &
+./marmot -config examples/node-1-config.toml -cluster-addr localhost:4221 -cluster-peers 'nats://localhost:4222/,nats://localhost:4223/' &
 job1=$!
 
 sleep 1
-./marmot -config examples/node-2-config.toml -cluster-addr localhost:4222 -cluster-peers 'nats://localhost:4221/' &
+./marmot -config examples/node-2-config.toml -cluster-addr localhost:4222 -cluster-peers 'nats://localhost:4221/,nats://localhost:4223/' &
```

Comment:
there is no point of passing `:4223` here because that node comes up in very last.

Reply:
Understand. But I think this optimization makes new user wondering why. Peer configuration symmertic to node 2 and node 3 is more intutive. Also, when you look at the JetStream doc, it's also using a symetric example. 
https://docs.nats.io/running-a-nats-service/configuration/clustering/jetstream_clustering

Indeed, I think specifying two having a different implication in term of HA.